platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/signed/SignedStateFileWriter.java

```diff
@@ -166,14 +175,40 @@ public static void writeSignedStateFilesToDirectory(
         writeSettingsUsed(directory, platformContext.getConfiguration());
 
         if (selfId != null) {
-            copyPreconsensusEventStreamFiles(
+            copyPreconsensusEventStreamFilesRetryOnFailure(
```

Comment:
It seems that if one file fails, we will retry all of them. It seems better to retry on every file individually. There might be issues if we try to copy the same files again. I will approve, up to you if you think this change should be made.